scripts/o3de/o3de/export_project.py

```diff
@@ -38,7 +38,7 @@
     # Test if Ninja is available from the command line to determine the generator and multi-config capability
```

Comment:
I know that this is not part of the PR, but this is a copy/paste error and the description does not match the action.
```suggestion
    # Test if Xcode is available from the command line to determine the generator
```